<launch>
	<node name="accelerometer" pkg="phidgets_ros" type="accelerometer.py" output="screen">
	    <!-- <param name="name" type="str" value="my_accelerometer"/> -->
		<param name="serial_number" type="int" value="93353"/>
		<param name="x_axis_id" type="int" value="0"/>
		<param name="y_axis_id" type="int" value="1"/>
		<param name="z_axis_id" type="int" value="2"/>
		<param name="sensitivity" value="0.0"/>
	</node>
</launch>